docs/develop/python/serving.mdx

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 11
+description: "Set up long-running services or Lambda handlers."
+---
+
+
+# Serving
+Restate services can run in a few ways: as an HTTP handler, or as an AWS Lambda handler.
+For both, you do the following:
+
+1. Create the app
+2. Bind one or multiple services to it.
+
+```python example.py
+CODE_LOAD::python/src/develop/serving.py#endpoint
+```
+
+The Python SDK follows the [ASGI](https://asgi.readthedocs.io/en/latest/introduction.html) standard for the serving of the services.
+The templates and examples use [Hypercorn](https://pypi.org/project/Hypercorn/) to serve the services.
+You can run them as follows:
+
+```shell
+python -m hypercorn --config hypercorn-config.toml -b localhost:9080 example:app
```

Comment:
Same comment as https://github.com/restatedev/documentation/pull/447#discussion_r1734892713, i don't think you need to specify the bind address here

Reply:
I think it's better to do so though since all our docs are written with the assumption that the service runs on 9080... Otherwise users will just have to look that up. I think it makes the experience just a bit more fluent.

Reply:
then how about you remove the hypercorn config? at least in the quickstart, it adds some nice that is not needed.

Reply:
Or can we add it to the hypercorn config? Then we can remove it from the other commands

Reply:
But the bind address is in the hypercorn config: https://github.com/restatedev/examples/blob/main/templates/python/hypercorn-config.toml#L1 that's why specifying that `-b` is redundant